r/tests/testthat/test-buffer.R

```diff
@@ -39,6 +39,7 @@ test_that("Buffer can be created from numeric vector", {
 })
 
 test_that("Buffer can be created from complex vector", {
+  skip("until cpp11 has a complex vector class")
```

Comment:
Is this something we need?

Reply:
Doing some archaeology, it was added specifically in https://github.com/apache/arrow/commit/13c63bdec9675f08b1631e0514dcf2a890d7e36b (https://issues.apache.org/jira/browse/ARROW-3823). No context in the PR or the JIRA to know if this was just added because it was possible or if it was required for something (that said, no evidence that it was demanded by some real use case).

Reply:
I don't think that we do need it.

Reply:
but just in case, I added it, we don't need a full implementation of `cpp11::complexs` for this